src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,121 @@
+package duke.main;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidTaskTypeException;
+import duke.task.TaskManager;
+import duke.task.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+import java.util.Scanner;
+
+public class Duke {
+    private static String separatingLine = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
```

Comment:
Consider making this a constant if it does not change. This can also be applied to other magic literals like "duke.txt", 0, 1, 100 etc.